update_rules_content.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Updates the ./tests/content/ok/ directory with the latest rules to test against
+
+function clean_up() {
+    rm -rf "$CLONE_TEMP_DIR"
+}
+trap clean_up EXIT
+
+RULES_REPO="https://gitlab.cee.redhat.com/ccx/ccx-rules-ocp.git"
+CONTENT_DIR="$( cd "$(dirname "$0")" && pwd )/tests/content"
```

Comment:
isn't `./tests/content` enough here?

Reply:
Or just `$(dirname "$0")`, no need to "cd" and "pwd", I guess

Reply:
@Sergey1011010 @JoseLSegura Can you tell me what are the advantages of using relative paths? I don't like them, so I use absolute paths whenever I can. 

`./tests/content` doesn't work when you call the script from another location, e.g. from the ok/ folder when I was testing it
`$(dirname "$0")` works when called from different location, but the outputted messages are less readable for me, e.g. `Attempting to clone repository into ../../../tests/content/tmp` instead of the readable absolute path

Reply:
Well, if you prefer absolute path I'm ok with that, but it's better to use `realpath` instead of "cd + pwd", IMHO.

In any case, you can merge from my point of view

Reply:
The readability of the code kinda suffers from using complicated/nested commands like this.

However, the much more pressing issue is why are we even putting real rule content into the `tests` directory? As far as understand, the rules are secret, but the tests directory is completely publicly visible. Or am I reading the script wrong and is that not what's happening here?

Reply:
I don't understand how is `"$( cd "$(dirname "$0")" && pwd )"` complicated or unreadable??
I can change it, but then the readability of the output is impacted. 

Regarding your second issue, I had only one idea how to solve and that would be to put it in a folder that would be in `.gitignore`, but I thought that testing against real rules would be a good idea instead of tailoring our mocked data. I didn't realize the content of the rules is to be kept secret, @tisnik can you weigh in on this issue pls?

Reply:
@Bee-lee Using `cd` and two levels of nested commands, which requires an unforunate number of quotation marks, certainly isn't the most readable code. However, that's a general problem with any shell script code. The alternative with `realpath` seems easier to comprehend to me.

Well, the rule content is not available anywhere publicly and there is no plan for it to be, so I'd say we definitely want to keep certain information about the rules and their content private, from what I've heard so far.

Reply:
@natiiix take a look now, i change it to realpath, changed the content directory and added it to .gitignore